Utilities/DataLoading/EdFi.SmokeTest.Console/Application/SmokeTestsConfiguration.cs

```diff
@@ -31,8 +31,12 @@ public class SmokeTestsConfiguration : IApiConfiguration, IApiMetadataConfigurat
 
         public int? SchoolYear { get; set; }
 
```

Comment:
Please address the next tasks related to the Smoke Test tool:

- Update ResolvedUrl method below to handle InstanceYearSpecific
- Since OAuthUrl also contains the instance template, call ResolvedUrl while initializing it (a couple of lines below).
- Initialize the InstanceId property (similar to what you did in BulkLoadClientConfiguration.cs).
- Define the short command line override for InstanceId (-i) [here](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/main/Utilities/DataLoading/EdFi.SmokeTest.Console/Application/CommandLineOverrides.cs#L48) (similar to what you did in [EdFi.BulkLoadClient.Console/Application/CommandLineOverrides.cs](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/589/files#diff-240ebfa575b97b4a1c1f400930838e0b4664a45e578eff259eb597d17c450fee)).

Reply:
@AxelMarquez Done